### PR TITLE
Missing the body

### PR DESCRIPTION
**Pull request by ys**
I changed the put signature but no specs failed :(

### CODE REVIEW

**Comment by atmos** [comment]
I think this is usually called method arity and type correctness.

**Comment by ys** [comment]
Yeah I screwed on this method arity change for sure :D

**Comment by ys** [comment]
I guess it wouldn't have happened if I was using Java and an editor lol

File: lib/escobar/heroku/app.rb [hunk 1]
```diff
@@ -54,7 +54,8 @@ def cache_key
 
       # Accepts either google authenticator or yubikey second_factor formatting
       def preauth(second_factor)
-        client.heroku.put("/apps/#{id}/pre-authorizations", second_factor).none?
+        client.heroku
+              .put("/apps/#{id}/pre-authorizations", "", second_factor).none?
```
Comment (atmos) [inline lib/escobar/heroku/app.rb]: new string for the http path?

**Pull request merged**